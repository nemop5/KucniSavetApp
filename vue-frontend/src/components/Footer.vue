<template>
    <!-- footer -->
	<footer class="footer">
		<div class="wrap">
			<span class="copy">&copy; 2019 Vega IT Sourcing</span>
		</div>
	</footer>
</template>

<script>
	export default {
  		name: 'Footer',
	}	
</script>
