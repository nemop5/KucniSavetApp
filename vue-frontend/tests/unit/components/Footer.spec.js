import { shallowMount } from '@vue/test-utils'
import Footer from '@/components/Footer.vue'

describe('Footer.vue', () => {
    test('footer snapshot', () => {
      const wrapper = shallowMount(Footer)
      expect(wrapper).toMatchSnapshot()
    })
  })