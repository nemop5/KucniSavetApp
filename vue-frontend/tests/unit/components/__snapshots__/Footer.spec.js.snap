// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer.vue renders span 2019 Vega IT Sourcing 1`] = `
<footer class="footer">
  <div class="wrap"><span class="copy">Â© 2019 Vega IT Sourcing</span></div>
</footer>
`;
